tests/zenko_tests/node_tests/backbeat/tests/retry/retry.js

```diff
@@ -35,78 +43,117 @@ function checkMetrics(prevBacklog, prevCompletions, prevFailures, prevPending,
-                cb),
-        next),
-        next => times(keys.length, (n, cb) =>
-            scalityUtils.waitWhilePendingCRR(srcBucket, keys[n], cb),
-        next),
-        next => times(keys.length, (n, cb) =>
-            scalityUtils.getHeadObject(srcBucket, keys[n],
-            (err, data) => {
-                if (err) {
-                   return cb(err);
-                }
-                const { ReplicationStatus, Metadata, VersionId } = data;
-                versionId = VersionId;
-                assert.strictEqual(ReplicationStatus, 'FAILED');
-                assert.strictEqual(
-                   Metadata[`${destFailLocation}-replication-status`],
-                   'FAILED');
-                setTimeout(function () {
-                    return cb();
-                }, 5000)
-            }),
-        next),
-        next => makeGETRequest('/_/backbeat/api/crr/failed',
+        next => awsUtils.deleteVersionedBucket(destFailBucket, next),
+        next => times(
+            keys.length,
+            (n, cb) => scalityUtils.putObject(
+                srcBucket,
+                keys[n],
+                Buffer.alloc(1),
+                cb,
+            ),
+            next,
+        ),
+        next => times(
+            keys.length,
+            (n, cb) => scalityUtils.waitWhilePendingCRR(srcBucket, keys[n], cb),
+            next,
+        ),
+        next => times(
+            keys.length,
+            (n, cb) => scalityUtils.getHeadObject(
+                srcBucket,
+                keys[n],
+                (err, data) => {
+                    if (err) {
+                        return cb(err);
+                    }
+                    const { ReplicationStatus, Metadata, VersionId } = data;
+                    // eslint-disable-next-line
+                    versionId = VersionId;
+                    assert.strictEqual(ReplicationStatus, 'FAILED');
+                    assert.strictEqual(
+                        Metadata[`${destFailLocation}-replication-status`],
+                        'FAILED',
+                    );
+                    return setTimeout(() => cb(), 5000);
+                },
+            ),
+            next,
+        ),
+        next => makeGETRequest(
+            '/_/backbeat/api/crr/failed',
             (err, res) => {
                 assert.ifError(err);
                 return getResponseBody(res, (err, body) => {
                     assert.ifError(err);
+                    // eslint-disable-next-line
                     postBody = JSON.stringify(body.Versions);
                     next();
                 });
-            }),
+            },
+        ),
         next => awsUtils.createVersionedBucket(destFailBucket, next),
-        next => makePOSTRequest('/_/backbeat/api/crr/failed', postBody,
+        next => makePOSTRequest(
+            '/_/backbeat/api/crr/failed',
+            // eslint-disable-next-line
```

Comment:
For my understanding: I take the linter disable lines we see throughout are band-aid fixes so as to not make significant code logic changes that would be required by the linter while working through linter issues?

Reply:
You're right, will change

Reply:
Changed in [a3ae3ff](https://github.com/scality/Zenko/pull/1528/commits/a3ae3ffeb90c2966771e6ba73fe4b8c485cbf4d2)